apps/react/src/features/anime/components/AnimeItem/AnimeItem.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, memo } from 'react';
+import { Avatar, ListItem, ListItemAvatar, ListItemText, Typography } from '@mui/material';
+
+import { Anime } from '@js-camp/core/models/anime';
+
+import styles from './AnimeItem.module.css';
+
+interface AnimeProps {
+
+  /** Anime. */
+  readonly anime: Anime;
+}
+
+const AnimeItemComponent: FC<AnimeProps> = AnimeProps => (
+  <ListItem className={styles['animeListItem']}>
```

Comment:
If you chosen to use camelCase for classes, you can omit square brackets
```suggestion
  <ListItem className={styles.animeListItem}>
```

Reply:
@artem-sotnichenko-saritasa There is an issue with ts/linting that doesn't allow to use it without square brackets.

Reply:
I helped @andrey-gazzaev to fix this, you can ask him